QUEST.tsv

```diff
@@ -522,210 +522,210 @@ QUEST_20150317_000521	Seems to be asleep.
+QUEST_20150317_000547	Welcome! {nl}You can buy great items for a cheap price if you browse carefully. {nl}Of course, it's also possible to spend a great sum on something that's supposed to be cheap…
 QUEST_20150317_000548	Proposal Card
-QUEST_20150317_000549	Roses that still have fragrance. {nl}They seem to have gone stale in time.
+QUEST_20150317_000549	Roses that still have fragrance. {nl}Though they seem to have withered with time.
 QUEST_20150317_000550	Benevolent Old Man
-QUEST_20150317_000551	When I was a kid, I heard that the demons were sealed in the Crystal Mines. {Nl}If that's true, then we are in a real dangerous situation now.
-QUEST_20150317_000552	I'm not sure if the mine should operate at a dangerous time like this. {Nl}Even if mining is our livelihood, a person's life is more important you know.
-QUEST_20150317_000553	I used to be a miner myself. {Nl}I hardly got to see any monsters back then, but I heard it's very different nowadays.
-QUEST_20150317_000554	They say there are many holy relics hidden in the Tenet Church. {Nl}I heard some things are said to move people to a place where no one can go… {Nl}Amazing isn't it?
-QUEST_20150317_000555	One day I will find out. {nl}Even if that entails me to meet the Goddess.
+QUEST_20150317_000551	When I was a kid, I heard that the demons were sealed in the Crystal Mine. {nl}If that's true, then we're in trouble.
+QUEST_20150317_000552	I'm not sure if the mine should be operating at a dangerous time like this. {nl}Even if mining is our livelihood, our lives are still more important.
+QUEST_20150317_000553	I used to be a miner myself. {nl}I didn't see many monsters back then, but I heard it's very different nowadays.
+QUEST_20150317_000554	They say there are many holy relics hidden in the Tenet Church. {nl}Some of them are said to transport people to a place where no one can go… {nl}Amazing, isn't it?
+QUEST_20150317_000555	One day, I will find out. {nl}Even if that entails my meeting the Goddess.
 QUEST_20150317_000556	I don't know what to do if you are like that…
 QUEST_20150317_000557	Crystal Mine 2nd Lot is closed until further notice due to safety reasons. {nl} - Klaipeda Mines Union
 QUEST_20150317_000558	Business Guide
 QUEST_20150317_000559	Welcome! This is the miner's resting hall. {nl}Warm potatoes and drinks have been prepared for you. {nl}{nl}Business hours: same as mining time - may change seasonally.
 QUEST_20150317_000560	Notice
-QUEST_20150317_000561	This is the Crystal Mines. {Nl}Infractions will be given and you will be forced to leave if you do not observe the following:
-QUEST_20150317_000562	1. Always wear a helmet and protective gear {nl}2. Do not enter restricted areas {nl}3. Never carry flamable objects {nl}4. If you see monsters, do not fight and file a report immediately.
+QUEST_20150317_000561	This is the Crystal Mine. {nl}You will be punished and forced to leave if you do not observe the following:
```

Comment:
This is the Crystal Mine. Violators of the rules below will be punished and forced to leave.